Warpi/line.py

```diff
@@ -0,0 +1,20 @@
+# Importing turtle module
+from turtle import *
+color("orange")
+bgcolor("black")
+# Drawing a line fwd 100 steps
+forward(100)
+# Turning left by 120 degrees
+left(120)
+# Drawing a line fwd 100 steps  
+forward(100)
```

Comment:
Nice comments explaining the code. I think you could put them to the right of the line of code they are explaining for better readability. 

```suggestion
forward(100) # Drawing a line fwd 100 steps

left(120) # Turning left by 120 degrees

forward(100) # Drawing a line fwd 100 steps  
```